exercises/practice/go-counting/go-counting.spec.js

```diff
@@ -0,0 +1,113 @@
+import { GoCounting } from './go-counting';
+
+describe('Go Counting', () => {
+  describe('getTerrytory', () => {
+    const board5X5 = ['  B  ', ' B B ', 'B W B', ' W W ', '  W  '];
```

Comment:
Currently, this constant would not be visible in the respective test runner output on the website. Since it is not very long, I would recommend to copy it into all the test cases that need it. Alternatively you can make this an arrow function that returns a fresh array every time. Otherwise, if a student manipulates this board by mistake, that can influence all the other tests that have this array as input.

Reply:
Agreed with June here.